core/solidus_core.gemspec

```diff
@@ -35,9 +35,9 @@ Gem::Specification.new do |s|
   s.add_dependency 'discard', '~> 1.0'
   s.add_dependency 'friendly_id', '~> 5.0'
   s.add_dependency 'kaminari-activerecord', '~> 1.1'
-  s.add_dependency 'monetize', '~> 1.1'
+  s.add_dependency 'monetize', '~> 1.8.0'
```

Comment:
Minor suggestion would be to add a comment as to why this is locked with a link to the issue that needs to be resolved until it can be relaxed again.

Reply:
Thanks for the suggestion, I've documented that in the commit message. I prefer not to add comments in the code and if we forgot and keep monetize locked to 1.8 is not that critical. If someone will need to update monetize to 1.9 a `git blame` will document the why of this. Does it make sense?